interop-cats/shared/src/main/scala/zio/interop/cats.scala

```diff
@@ -101,9 +101,16 @@ abstract class CatsInstances extends CatsInstances1 {
     new CatsMonoidK[R, E]
 
   implicit def bifunctorInstance[R]: Bifunctor[ZIO[R, *, *]] =
-    new CatsBifunctor[R] {}
+    new CatsBifunctor[R]
 
-  implicit def zioArrowInstance[E]: ArrowChoice[ZIO[*, E, *]] = new CatsArrow[E]
+  implicit val rioArrowInstance: ArrowChoice[RIO] =
+    new CatsArrow[Throwable]
+
+  implicit val urioArrowInstance: ArrowChoice[URIO] =
+    new CatsArrow[Nothing]
+
+  implicit def zioArrowInstance[E]: ArrowChoice[ZIO[*, E, *]] =
```

Comment:
@joroKr21 
I'd pull this down into a low priority trait because the choice for `ZIO[*, Throwable, *]` even if not ambiguous now due to specifics, is ambiguous in actuality and may become ambiguous if implicits are improved in the future.

Reply:
Sure, will do. 

Note however that I wouldn't say more ambiguity would be an improvement. If I wrote an implicit for some concrete type by hand, I want it to be preferred over the generic one.

Reply:
@joroKr21 
Hmm, seems like dotty/scala already behaves like that https://scastie.scala-lang.org/Xa5gdVA5RDOKTEyF5Bob3A https://scastie.scala-lang.org/LoIvb5yiR42lbyPCZMSdLA
So no reason for LP trait here